readthedocs/settings/base.py

```diff
@@ -101,6 +101,16 @@ class CommunityBaseSettings(Settings):
     RTD_MAX_CONCURRENT_BUILDS = 4
     RTD_BUILD_STATUS_API_NAME = 'docs/readthedocs'
 
+    # String contents to store in each virtualenv/pip.conf, useful to set a
+    # custom index-url.
```

Comment:
Instead of mentioning "Useful..." in these comments, I would just link the documentation of the pip.conf file here.